src/main/java/Objects/Packets/SYN.java

```diff
@@ -2,9 +2,11 @@
 
 public class SYN extends Packet {
 
-    public SYN() {
-        /*setCost(1);
-        setStealth(1);*/
+    public SYN(final String team) {
+        setPacketType("SYN");
+        setTeam(team);
+        setCost(1);
+        setStealth(1);
     }
 
```

Comment:
what is this space?

Reply:
1/10th of the room you need for your thumb